src/nile/core/call_or_invoke.py

```diff
@@ -34,22 +37,37 @@ def call_or_invoke(
     else:
         address, abi = next(deployments.load(contract, network))
 
-    address = hex_address(address)
-    arguments = [
+    args = set_args(network)
+    command_args = set_command_args(
+        inputs=params, signature=signature, max_fee=max_fee, query_flag=query_flag
+    )
+
```

Comment:
probably not now, but i wonder if we could abstract this into the starknet_cli module i'm thinking about